game-core/src/main/java/games/strategy/engine/posted/game/pbf/NodeBbForumPoster.java

```diff
@@ -66,10 +59,7 @@
   private NodeBbForumPoster(final ForumPostingParameters forumPostingParameters) {
     this.topicId = forumPostingParameters.topicId;
     this.forumUrl = forumPostingParameters.forumUrl;
-    this.username =
-        Arrays.withSensitiveArrayAndReturn(() -> forumPostingParameters.username, String::new);
-    this.password =
-        Arrays.withSensitiveArrayAndReturn(() -> forumPostingParameters.password, String::new);
+    this.token = new String(forumPostingParameters.token);
```

Comment:
Codacy found an issue: [Avoid instantiating String objects; this is usually unnecessary.](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5619573)